documentation/docs/04_step_by_step.md

```diff
@@ -57,7 +57,9 @@ Send a regular amount from the main account to the savings address.
 $ ./wallet main
 > Account "main": send [ADDRESS] 1000000
 > ...
-> INFO  Transaction created ...
+> INFO  Transaction sent:
+> transaction id: 0x...
```

Comment:
I promise this is my last suggestion but I just realised this: It's a bit weird to me that we pretty print the BlockId but not the TransactionId 😬

Reply:
Yeah because it's an option, I guess some `.map(|id| id.to_string()).unwrap_or("no block id?".to_string())` or something could work, should I do that?

Reply:
Maybe the opposite is fine, pretty printing the TransactionId, we do that in most of the CLI wallet already I think

Reply:
Sorry, where do you see pretty printing? It's `transaction id: {}\n{:?}"` everywhere